tests/test_errors.py

```diff
@@ -0,0 +1,36 @@
+from n2y.errors import APIErrorCode
+
+
+def test_apierrorcode_contains():
+    errors = [
```

Comment:
Very cool! Checkout pytest fixtures for another way to do this: https://docs.pytest.org/en/6.2.x/fixture.html#parametrizing-fixtures

Reply:
Yep yep! We use them in other tests in the notion-qms repo I think. If we expand this module I'll probably end up shifting to that method to avoid definition repetition.